dotnet/DDB.Tests/Tests.cs

```diff
@@ -59,7 +59,9 @@ public void testAddRemove()
             Assert.AreEqual(entries.Count, 2);
 
             DroneDB.Remove("testAdd", Path.Join("testAdd", "file.txt"));
-            Assert.Throws<DDBException>(() => DroneDB.Remove("testAdd", "invalid"));
+            
+            // NOTE: Should we throw an exception when the requested file to remove does not exist?
```

Comment:
Yes.

If a file is not in the index, and a user attempts to remove it, it should throw an error.

```
[piero:/data/test] $ touch test.txt
[piero:/data/test] $ git add test.txt
[piero:/data/test] $ git rm --cached test.txt
rm 'test.txt'
[piero:/data/test] $ git rm --cached test.txt
fatal: pathspec 'test.txt' did not match any files
[piero:/data/test] 128 $ 
```

A bit like git.

Reply:
Ok I will add this behaviour to the `ddb remove` command

Reply:
Fixed in 14d6103